app/components/post_body/post_body.test.js

```diff
@@ -9,6 +9,7 @@ import PostBodyAdditionalContent from 'app/components/post_body_additional_conte
 import {shallowWithIntl} from 'test/intl-test-helper';
 
 import PostBody from './post_body.js';
+import * as SystemMessageHelpers from './system_message_helpers';
```

Comment:
see above comment about the file location

Reply:
(Response in previous comment conversation)